color/src/parse.rs

```diff
@@ -451,21 +451,24 @@ impl<'a> Parser<'a> {
     }
 }
 
-// Arguably this should be an implementation of FromStr.
-/// Parse a color string in CSS syntax into a color.
+/// Parse a color string prefix in CSS syntax into a color.
+///
+/// Returns the byte offset of the unparsed remainder of the string and the parsed color. See also
+/// [`parse_color`].
 ///
 /// # Errors
 ///
 /// Tries to return a suitable error for any invalid string, but may be
 /// a little lax on some details.
-pub fn parse_color(s: &str) -> Result<DynamicColor, ParseError> {
+pub fn parse_color_with_remainder(s: &str) -> Result<(usize, DynamicColor), ParseError> {
```

Comment:
Or `fn parse_color_prefix`, or another name.

Reply:
I like the shorter name, I think it's clear enough.